### PR TITLE
.net 8 upgrade

### PR DESCRIPTION
**Pull request by PradeepVoorukonda**

### CODE REVIEW

**Review by abu-khubaib — COMMENTED** [review]
Following settings are missing in azure-pipelines.yml file
GitVersionVersion: 5.12.x
DotNetCoreVersion: 8.x

**Review by abu-khubaib — DISMISSED** [review]

File: NCS.DSS.ContentEnhancer.Tests/NCS.DSS.ContentEnhancer.Tests.csproj [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+    <IsTestProject>true</IsTestProject>
+  </PropertyGroup>
```
Comment (bmatthews-DFE) [inline NCS.DSS.ContentEnhancer.Tests/NCS.DSS.ContentEnhancer.Tests.csproj:10]: Confirm if all attributes in this group are necessary, as they differ to other .NET 8 repos
@amjones99 @PradeepVoorukonda

File: NCS.DSS.ContentEnhancer/NCS.DSS.ContentEnhancer.csproj [hunk 2]
```diff
@@ -1,17 +1,23 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
     <AzureFunctionsVersion>v4</AzureFunctionsVersion>
+    <OutputType>Exe</OutputType>
+    <ImplicitUsings>enabled</ImplicitUsings>
```
Comment (bmatthews-DFE) [inline NCS.DSS.ContentEnhancer/NCS.DSS.ContentEnhancer.csproj:6]: Replace `<ImplicitUsings>enabled</ImplicitUsings>` with `<ImplicitUsings>enable</ImplicitUsings>`

Comment (amjones99) [reply NCS.DSS.ContentEnhancer/NCS.DSS.ContentEnhancer.csproj]: Resolved

**Review by amjones99 — APPROVED** [review]

**Pull request merged**